cmd/cp-main.go

```diff
@@ -459,6 +508,7 @@ func mainCopy(ctx *cli.Context) error {
 		sseKeys = key
 	}
 	sse := ctx.String("encrypt")
+	userMetadata := ctx.String("attr")
```

Comment:
We should always parse the input here, i.e parseMetadata should be used here.. look at checkCopySyntax or getEncKeys for adding validations..